Packs/FeedIntel471/Integrations/Intel471Indicator/Intel471Indicator.yml

```diff
@@ -134,25 +134,32 @@ configuration:
   name: insecure
   required: false
   type: 8
+- defaultvalue: 'true'
+  additionalinfo: Create relationships between indicators as part of Enrichment.
+  display: Create relationships
+  name: create_relationships
+  required: false
+  type: 8
 description: |-
   "Intel471's Malware Intelligence is focused on the provisioning of a high fidelity and timely indicators feed with rich context, TTP information, and malware intelligence reports.
   This feed allows customers to block and gain an understanding of the latest crimeware campaigns and is for those that value timeliness, confidence (little to no false positives), and seek rich context and insight around the attacks they are seeing."
-display: Intel471 Malware Feed
-name: Intel471 Malware Feed
+display: Intel471 Indicator Feed
+name: Intel471 Indicator Feed
 script:
   commands:
   - arguments:
     - default: false
       defaultValue: '50'
-      description: The maximum number of results to return.
+      description: |-
+        The maximum number of results to return. Will limit the result for each indicator type. Default is 50.
       isArray: false
       name: limit
       required: false
       secret: false
     deprecated: false
     description: Gets the feed indicators.
     execution: false
-    name: intel471-malware-get-indicators
+    name: intel471-indicators-get-indicators
```

Comment:
let's keep the command name.
you can add another one that will follow the same one and deprecate this one if you wish.
this way you will not break BC.